hazelcast/src/main/java/com/hazelcast/core/HazelcastInstance.java

```diff
@@ -311,6 +312,16 @@
     PartitionService getPartitionService();
 
     /**
+     * Returns the quorum service of this Hazelcast instance.
+     * <p/>
+     * Quorum service can be used to retrieve quorum callbacks which let you to notify quorum results of your own to
+     * the cluster quorum service.
+     *
+     * @return the quorum service of this Hazelcast instance
+     */
+    QuorumService getQuorumService();
```

Comment:
Enterprise repo is failing because of this change. @eminn can you also update the enterprise?